test/basic-querying.test.js

```diff
@@ -172,7 +177,7 @@ describe('basic-querying', function() {
       User.find({skip: 1, limit: 4, order: 'seq'}, function(err, users) {
         should.exists(users);
         should.not.exists(err);
-        users[0].seq.should.be.eql(1);
+        if (!isCassandraConnector) users[0].seq.should.be.eql(1);
```

Comment:
As the immediate next step, added "bdd.itIf(!isCassandraConnector," for the cases that contain un-supported filtering conditions.  Then, will look at "connectorCapabilities"